### PR TITLE
10.0 added country manufacturer

### PR DESCRIPTION
**Pull request by ovnicraft**

### CODE REVIEW

**Comment by moylop260** [comment]
could you check red ci result, please?

File: product_manufacturer/__manifest__.py [hunk 1]
```diff
@@ -19,17 +19,14 @@
...
-    'data': [
-        'views/product_manufacturer_view.xml'
-    ],
-    'auto_install': False,
-    'installable': True,
+    "name": "Product Manufacturer",
+    "version": "10.0.1.1.0",
+    "summary": "Adds manufacturers and attributes on the product view.",
+    "author": "OpenERP SA, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```
Comment (nikul-serpentcs) [inline product_manufacturer/__manifest__.py:26]: @ovnicraft Add `website` Key

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

**Comment by ovnicraft** [comment]
@moylop260 its 79 cols warning on code.

**Comment by ovnicraft** [comment]
Travis is not related to this PR

**Pull request closed without merging**